docs/components/pagination/Example.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+<div class="component" ref="el" />
+</template>
+
+<script setup>
+import { createElement } from 'react'
+import { createRoot } from 'react-dom/client'
+import { ref, onMounted } from 'vue'
+import { Pagination } from '@warp-ds/react'
+
+const el = ref()
+
+let paginationData = ref({
+    currentPage: 0,
+});
+onMounted(() => {
+  const root = createRoot(el.value)
+  root.render(createElement(Pagination, { currentPage: paginationData.value.currentPage,
+            numPages: 3,
+            lastPage: 3,
+            createHref: (page) => `?page=${page}`,
+            onChange: (page) => paginationData.value.currentPage++ }, null))
```

Comment:
Does anyone know how to increment a counter in vue? @magnuh @felicia-haggqvist @BalbinaK

Reply:
I tried to play around with it but got nowhere. Very curious what's the solution here 👀

Reply:
Yeah same here! 🙈 Are we sure that the onChange is being triggered in the react Pagination component?

Reply:
I did manage to console log the correct value of `paginationData.value.currentPage` inside the `onChange`:
```
onChange: (page) => {
          paginationData.value.currentPage = page;
          console.log(`hello ${page} ${paginationData.value.currentPage}`); // hello 1 1 ... hello 2 2 ... hello 3 3
        }
```
 
 but the ref value doesn't seem to be updated outside. As though the Vue component didn't re-render.